src/v/cloud_io/remote.cc

```diff
@@ -256,7 +258,7 @@ ss::future<upload_result> remote::upload_stream(
         }
         auto fut = co_await ss::coroutine::as_future(
           _pool.local().acquire_with_timeout(
-            fib.root_abort_source(), fib.get_timeout(), fib()));
+            fib.root_abort_source(), _lease_timeout(), fib()));
```

Comment:
is this the overwriting the change that was made just two commits back?

Reply:
eh, i guess the API change also changes the logic in `acquire_with_timeout` so maybe it's a wash